src/brpc/server.cpp

```diff
@@ -123,6 +123,8 @@ const int INITIAL_CERT_MAP = 64;
 // compilation units is undefined.
 const int s_ncore = sysconf(_SC_NPROCESSORS_ONLN);
 
+DEFINE_uint64(latency_scale_factor, 1, "latency scale factor, used by method status, etc., latency_us = latency * latency_scale_factor");
```

Comment:
I think this gflag can be directly defined in `latency_recorder.cpp`, so that it doesn't need to be passed through so many layers of parameters.

Reply:
ok

Reply:
@wwbmmm done